### ISSUE TITLE
If PreferPrivateSession is specified in the signinwebUI options, the modal can be closed.

### ISSUE THREAD

**Issue by reolch** [aws-amplify/amplify-ios#1674]
### Describe the bug

We apologize for the inconvenience. We are using machine translations, so the content may not be conveyed correctly.

I am reproducing this issue only with iPhoneSE 2nd Gen.
As noted in the title, if PreferPrivateSession is specified in the signinwebUI method options, it is possible to close the authentication modal displayed by the method.

### Steps To Reproduce

```swift
Steps to reproduce the behavior:
1. I implemented it according to the following document provided by AWS.
* https://aws.amazon.com/jp/getting-started/hands-on/build-ios-app-amplify/module-three/
2. Add preferPrivateSession to the options of Amplify.Auth.signInWithWebUI method
3. Press the sign-in button
4. Swipe down on the presented modal to close the modal
```

### Expected behavior

The presented modal should not be closed even if the option preferPrivateSession is specified.

### Amplify Framework Version

1.19.2

### Amplify Categories

Auth

### Dependency manager

Cocoapods

### Swift version

5.0

### CLI version

-

### Xcode version

12.5.1

### Is this a regression?

No

### Device

iPhoneSE 2nd Gen

### iOS Version

iOS 13.6

### Specific to simulators
It is also reproduced in the simulator.
Reproduce it on the actual machine.

**Comment by ameter** [comment]
Thanks for reporting this.  We will investigate this issue and provide an update here when we have additional information.

**Comment by royjit** [comment]
`preferPrivateSession` refers to whether to use a private session or not and it is not related to the UI presentation of the modal. Swipe down feature is provided as default by Apple to cancel the signIn flow if user doesnot want to signIn. 

What is the use case to not allow swipe down cancel in signinwebUI?

**Comment by harsh62** [comment]
The [Amplify code](https://github.com/aws-amplify/amplify-ios/blob/main/AmplifyPlugins/Auth/AWSCognitoAuthPlugin/Dependency/AuthenticationProviderAdapter%2BSignIn.swift#L289)  has been setup correctly to not accept model events outside of the view bounds according to the
[Apple Documentation](https://developer.apple.com/documentation/uikit/uiviewcontroller/3229894-ismodalinpresentation). Further the iOS documentation states that as follows: 
```
    // modalInPresentation is set on the view controller when you wish to force the presentation hosting the view controller into modal behavior. When this is active, the presentation will prevent interactive dismiss and ignore events outside of the presented view controller's bounds until this is set to NO.
    @available(iOS 13.0, *)
    open var isModalInPresentation: Bool
```
Closing the ticket because the team has no actions it can take on the ticket. 

Workaround is to suggest the client to update to the latest version of iOS.

**Comment by reolch** [comment]
Thank you for your response.
We do not believe that the behavior should change when the preferPrivateSession option is specified or not specified.
